.pipelines/npm/npm-conformance-tests.yaml

```diff
@@ -74,18 +74,16 @@ stages:
             arch: amd64
             name: npm
             os: linux
-          npm_windows2022_amd64:
+          npm_windows_amd64:
             arch: amd64
             name: npm
             os: windows
-            os_version: ltsc2022
       steps:
         - template: ../containers/container-template.yaml
           parameters:
             arch: $(arch)
             name: $(name)
             os: $(os)
-            os_version: $(os_version)
```

Comment:
NPM only supports 2022. Is this change a cause for concern in that regard?

Reply:
it should not be, 2022 is still able to pull this image